roscpp_azure_iothub/src/ros_azure_iothub_cpp_node.cpp

```diff
@@ -244,25 +292,56 @@ void topicCallback(const topic_tools::ShapeShifter::ConstPtr& msg,
     parser.deserializeIntoFlatContainer( topic_name, Span<uint8_t>(buffer), &flat_container, 100);
     parser.applyNameTransform( topic_name, flat_container, &renamed_values );
 
-    ROS_INFO("--------- %s ----------", topic_name.c_str() );
-    sendMsgToAzureIoTHub(topic_name.c_str(), deviceHandle);
-    for (auto it: renamed_values)
+    // Send info to IoTHub via reported properties 
+    if (!(std::find(topicsToReport.begin(), topicsToReport.end(), topic_name.c_str()) == topicsToReport.end()))
     {
-        const std::string& key = it.first;
-        const Variant& value   = it.second;
-        char _buffer [256] = {0};
-        snprintf(_buffer, sizeof(_buffer), " %s = %f", key.c_str(), value.convert<double>());
-        ROS_INFO("%s",_buffer);
-        sendMsgToAzureIoTHub(_buffer, deviceHandle);
+        ROS_INFO("Sending message from %s to IoTHub via reported properties ", topic_name.c_str()); 
```

Comment:
might be better as ROS_DEBUG, since this is likely to occur frequently and really only useful in debugging.

Reply:
Ok I'll update that!